hooks/post_gen_project.py

```diff
@@ -324,6 +312,34 @@ def remove_storages_module():
     os.remove(os.path.join("{{cookiecutter.project_slug}}", "utils", "storages.py"))
 
 
+def handle_licenses():
```

Comment:
Is this to remove the Jekyll front matter? I'm wondering if wouldn't make more sense to do this work in the `update_licenses.py` scripts maybe? 

We can afford to use more modern Python there, we control the Python version better.

Reply:
Sorta, we need the Jekyll formatting *at first*. Each file's name is not the same as the title. So in order to find the correct file, we loop through each file until we find one that has the Jekyll title of the exact license name from cookiecutter.json.

We could improve on this inefficient method though by having `update_licenses.py` update this Python file to have a dictionary that maps titles (the dict keys) to their files (the dict values). That way we can remove the need for the Jekyll headers.

Reply:
Ah, I see...